src/components/DeckMechanisms/index.js

```diff
@@ -42,6 +42,7 @@ const getDefaultState = props => ({
     activeDawnsparks: 0,
     noUnitsOnFirstTurn: true,
     frozenEnemiesLevel: 0,
+    emptyCellsIndicator: 4,
```

Comment:
I feel like we might be overthinking this? Rain of Frogs, regardless of the level, will fill the row. Maybe we just have a boolean called `baselocked`?